internal/cmdrender/executor.go

```diff
@@ -375,25 +372,45 @@ func trackOutputFiles(hctx *hydrationContext) error {
 }
 
 // detectPathConflicts returns an error if the same index/path is on multiple resources
-func detectPathConflicts(nodes []*yaml.RNode) error {
-	// map has structure path -> index -> bool
+// if there are nodes that are exact duplicates of each other, it removes the duplicate
+func detectPathConflicts(nodes []*yaml.RNode) ([]*yaml.RNode, error) {
+	// map has structure path -> index -> RNode
 	// to keep track of paths and indexes found
-	pathIndexes := make(map[string]map[string]bool)
+	pathIndexes := make(map[string]map[string]*yaml.RNode)
+	var result []*yaml.RNode
+
 	for _, node := range nodes {
 		fp, index, err := kioutil.GetFileAnnotations(node)
 		if err != nil {
-			return err
+			return nil, err
 		}
 		fp = path.Clean(fp)
 		if pathIndexes[fp] == nil {
-			pathIndexes[fp] = make(map[string]bool)
+			pathIndexes[fp] = make(map[string]*yaml.RNode)
 		}
-		if _, ok := pathIndexes[fp][index]; ok {
-			return fmt.Errorf("resource at path %q and index %q already exists", fp, index)
+		if match, ok := pathIndexes[fp][index]; ok {
+			// path conflict, check if resource has the same content
+			if !duplicateNode(match, node) {
+				return nil, fmt.Errorf("resource at path %q and index %q already exists", fp, index)
+			}
+		} else {
+			result = append(result, node)
 		}
-		pathIndexes[fp][index] = true
+		pathIndexes[fp][index] = node
 	}
-	return nil
+	return result, nil
+}
```

Comment:
I am not able see the connection of this change with `fn render being able to create resource in parent` ? Can you elaborate more on that ?

Reply:
Maybe we can have a longer discussion about this since this doesn't seem like the right approach for this issue. 

Currently, fn render will create a duplicate of the resource when the path has prefix `../`. I think it is something related to being added to the input once as a transitive resource when processing the subpackage, and again when collecting all the resources in the current package. 

The fix here is to remove duplicates before outputting the resources. 

An alternate (and probably better) approach that I can work on is trying to make it not duplicate the resource in the first place, but that one will take a bit more time and investigation.

Reply:
> I think it is something related to being added to the input once as a transitive resource when processing the subpackage, and again when collecting all the resources in the current package.

You are right, we should root cause this issue that will help us address this correctly.

Reply:
Closing this PR. I'll open a new one after further investigation.